packages/client/tailwind.preset.js

```diff
@@ -0,0 +1,157 @@
+/**
```

Comment:
This file can be published to npm and used in other places where we use tailwind as a [preset](https://tailwindcss.com/docs/presets), for example parabol.co, icebreakers.parabol.co